requirements.txt

```diff
@@ -1 +1,3 @@
 Django>=4.0,<5.0
+psycopg2>=2.9.5,<3.0
+djangorestframework>=3.14.0,<4.0
```

Comment:
It's a good idea to list the exact versions of the packages in the `requirements.txt` file. It's easier to recreate the environment and avoid compatibility issues 👍🏼